### DISCUSSION TITLE
How many columns can Hydra handle?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [hydradatabase/hydra#140] [Q&A] [answered]
Hi Hydra team, I am wondering what is the limit of the number of columns in Hydra?

Thanks! 😃

**Comment by wuputah** [comment, marked as answer]
Hey there! We are bound by the limits of Postgres:
https://www.postgresql.org/docs/current/limits.html

So we currently support up to 1600 columns per table (relation), though we haven't tested a large number of columns, so I would be curious if you run into any issues with large numbers of columns.

**Reply by hongbo-miao** [reply]
Thanks @wuputah for the info! That is also the one I found. If I found not accurate, I will post here. ☺️
